lib/negroku/templates/tasks/eye/eye.rb.erb

```diff
@@ -10,8 +10,8 @@ Eye.application '<%= fetch(:application)%>' do
   # except `env`, which merging down
 
   working_dir File.expand_path(File.join(File.dirname(__FILE__), %w[ processes ]))
```

Comment:
well, now I think we should stick to `:current_path` I think the eye configuration is loaded on `load` and used at `restart`, I think changing the target for the `:current_path` symlink is not going to be a problem